### DISCUSSION TITLE
How to filter by many to many relation?

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#3152] [Archive]
Sorry to ask this again, but can't find how can I filter by m2m relation, like:

```
[accounts][id][_eq]': account.id
```

is smth like this supported?

**Comment by benhaynes** [comment]
@rijkvanzanten — is this possible?

**Reply by rijkvanzanten** [reply]
@botzill Do you have an example of tables and what you'd like to filter by?

I think your example is missing the junction table in the path, try

`filter[accounts][account_id]=15` and/or
`filter[accounts][account_id][first_name]=rijk`
